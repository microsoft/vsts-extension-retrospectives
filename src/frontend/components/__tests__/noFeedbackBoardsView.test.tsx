import React from 'react';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";
import NoFeedbackBoardsView, { NoFeedbackBoardsViewProps } from '../noFeedbackBoardsView';

const mockOnCreateBoardClick = jest.fn(() => { });

const defaultTestProps: NoFeedbackBoardsViewProps = {
  onCreateBoardClick: mockOnCreateBoardClick
}

describe('No Feedback Boards View component', () => {
  it ('renders correctly.', () => {
    const wrapper = shallow(<NoFeedbackBoardsView {...defaultTestProps} />);
    const component = wrapper.children().dive();
    expect(toJson(component)).toMatchSnapshot();
  });
});