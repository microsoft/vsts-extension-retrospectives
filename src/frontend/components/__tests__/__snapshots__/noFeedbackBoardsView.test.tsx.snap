// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Feedback Boards View component renders correctly. 1`] = `
<div
  className="no-boards-container"
>
  <div
    className="no-boards-text"
  >
    Get started with your first Retrospective
  </div>
  <div
    className="no-boards-sub-text"
  >
    Create a new board to start collecting feedback and create new work items.
  </div>
  <CustomizedDefaultButton
    className="create-new-board-button"
    onClick={[MockFunction]}
    primary={true}
    text="Create Board"
  />
</div>
`;
